src/main/java/rs117/hd/data/materials/Material.java

```diff
@@ -225,7 +225,10 @@
 
 	DIRT_1,
 	DIRT_2,
-	GRAVEL,
+	GRAVEL_N,
```

Comment:
I changed the gravel texture with one i think is far superior; If we don't like it we can scrap them.
Scroll to the bottom to see:
https://github.com/117HD/RLHD/pull/102/commits/98c4b036fa144364e9cbca9eec6d8f79fa532ba6

Reply:
Looking at this side by side, I think I might have to agree that it's superior. This might just be some innate resistance to change on my part :sweat_smile: It certainly feels more natural, but maybe less characteristic?

![image](https://user-images.githubusercontent.com/831317/201412530-a9f6298e-5542-4bd6-90f7-07741ddc4566.png)
![image](https://user-images.githubusercontent.com/831317/201412549-96066a9f-9668-4cbe-a1ba-d152c58f1958.png)
![image](https://user-images.githubusercontent.com/831317/201412587-62c5240c-b851-4289-8777-c76d3697ca37.png)
![image](https://user-images.githubusercontent.com/831317/201412630-87d60810-4ca8-4696-818f-93239e8c02d4.png)

The main part lacking I'd say is detail up close, which might be possible to achieve by tuning specularity and adding a normal map. That's definitely worth a shot I'd say.

Reply:
I'm fairly happy with how the gravel looks at this point, but it might be worth considering keeping the old texture available as a toggle, since this changes a core texture that's seen in a lot of places.
Updated screenshots:
![image](https://user-images.githubusercontent.com/831317/202064239-882f4c7c-d1df-4767-8f75-e60690dcdcbe.png)
![image](https://user-images.githubusercontent.com/831317/202064277-7ced9b13-73dd-47dc-96bd-1802eb50d049.png)

Reply:
How would i go about making a toggle for said texture?

Reply:
Added a toggle for this in https://github.com/117HD/RLHD/pull/102/commits/aef1e617337144022f173d7c4db5331f744b94e0